modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/ServerImpl.java

```diff
@@ -937,8 +937,11 @@ private boolean pingNode(TcpDiscoveryNode node) {
 
                         if (IgniteSpiOperationTimeoutHelper.checkFailureTimeoutReached(e)
                             && (spi.failureDetectionTimeoutEnabled() || timeout != 0)) {
-                            log.warning("Failed to ping node [nodeId=" + nodeId + "]. Reached the timeout " +
-                                (timeout == 0 ? spi.failureDetectionTimeout() : timeout) + "ms. Cause: " + e.getMessage());
+                            if (log.isDebugEnabled()) {
```

Comment:
1. pingNode can be called not only by cleanIpFinder, currently spams only cleanIpFinder method, but for other calls information about error is helpful and we shouldn't disable it.
2. Only timeout error is fixed, but there can be other causes. Next time pingNode can spam log again, but with another error message.
3. What about test?

Reply:
@alex-plekhanov thank you, I agree with you and have made changes to the PR after your comments